tests/core/test_editor.py

```diff
@@ -0,0 +1,79 @@
+import wiki.editors
+from django.forms import Textarea
+from wiki.editors.base import BaseEditor
+
+from ..base import RequireRootArticleMixin
+from ..base import WebTestBase
+from ..base import wiki_override_settings
+
+
+class CustomEditor(BaseEditor):
+    def get_widget(self, revision=None):
+        return Textarea(attrs={"data-revision": revision.pk})
+
+    def get_admin_widget(self, revision=None):
+        return Textarea(attrs={"data-revision": revision.pk})
```

Comment:
Very nice editor, requires no JS :)

Reply:
> requires no JS 

Actually true (I'm trying to use CodeMirror, which is ES6) :stuck_out_tongue_winking_eye: